SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -880,6 +880,18 @@ public List<Person> GetPersonsByListOfIds(List<long> ids)
             return _databaseContext.Persons.Where(x => ids.Contains(x.Id)).ToList();
         }
 
+        public Person GetPersonWithPersonalRelationshipsByPersonId(long personId)
+        {
+            var personWithRelationships = _databaseContext
+                .Persons
+                .Include(person => person.PersonalRelationships)
+                .FirstOrDefault(p => p.Id == personId);
+
+            personWithRelationships.PersonalRelationships = personWithRelationships.PersonalRelationships.Where(pr => pr.EndDate == null).ToList();
```

Comment:
I wonder if there should be a parameter that would determine whether only active relationships should be returned?

Reply:
We can add one and default it to only active ones for now and if we ever need it we can always switch it in the use case.

Reply:
I've added a `includeEndedRelationships` to the method and set it `false` as the default.